### PR TITLE
Es 7 11

### PR DESCRIPTION
**Pull request by worleydl**

### CODE REVIEW

File: src/main/java/com/o19s/es/ltr/action/TransportListStoresAction.java [hunk 1]
```diff
@@ -61,20 +61,14 @@
...
     @Inject
-    public TransportListStoresAction(Settings settings, TransportService transportService,ClusterService clusterService,
-                                     ThreadPool threadPool, ActionFilters actionFilters,
-                                     IndexNameExpressionResolver indexNameExpressionResolver, Client client) {
+    public TransportListStoresAction(Settings settings, TransportService transportService,
+                                     ClusterService clusterService, ThreadPool threadPool, ActionFilters actionFilters,
+                                     IndexNameExpressionResolver indexNameExpressionResolver, String executor, Client client) {
         super(ListStoresAction.NAME, transportService, clusterService, threadPool,
-            actionFilters, ListStoresActionRequest::new, indexNameExpressionResolver);
+            actionFilters, ListStoresActionRequest::new, indexNameExpressionResolver, ListStoresActionResponse::new, ThreadPool.Names.SAME);
```
Comment (worleydl) [inline src/main/java/com/o19s/es/ltr/action/TransportListStoresAction.java:68]: We may want to use the new `executor` param.

Comment (nathancday) [reply src/main/java/com/o19s/es/ltr/action/TransportListStoresAction.java:68]: Whoops didn't mean to resolve this, I has the swapped in for ThreadPool at some point, but I will put `executer` back in there tonight

Comment (worleydl) [reply src/main/java/com/o19s/es/ltr/action/TransportListStoresAction.java:66]: I tried passing it thru and it caused tests to fail out so I'm sticking with the SAME for now.

**Comment by worleydl** [comment]
I think this is ready for prime time.  We may want to do some cleaning of the commented out lines in build.gradle but otherwise 👍

**Review by nathancday — APPROVED** [review]
Damn that green check feels good!

**Pull request merged**